### PR TITLE
Update nifi

### PR DESCRIPTION
**Pull request by daniel-lau-mf**

### CODE REVIEW

File: helm/idol-nifi/resources/flow-basic-idol.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
                             "TangibleCharacters": null,
                             "RedactedOutput": "false",
                             "TableCellValuePath": "\".\",\"region/text\"",
-                            "ResourceFiles": "/opt/nifi/nifi-current/eduction/pii/combined_*.ecr",
+                            "ResourceFiles": "/opt/nifi/nifi-current/eduction/pii/combined_name.ecr,/opt/nifi/nifi-current/eduction/pii/combined_date.ecr,/opt/nifi/nifi-current/eduction/pii/combined_driving.ecr,/opt/nifi/nifi-current/eduction/pii/combined_health.ecr,/opt/nifi/nifi-current/eduction/pii/combined_national_id.ecr,/opt/nifi/nifi-current/eduction/pii/combined_passport.ecr,/opt/nifi/nifi-current/eduction/pii/combined_telephone.ecr,/opt/nifi/nifi-current/eduction/pii/combined_tin.ecr",
```
Comment (lewis-mackay-mf) [inline helm/idol-nifi/resources/flow-basic-idol.json:79]: Is it worth making this configurable from `values.yaml`? Having a sensible set of defaults in that file might be useful for interested users.

File: helm/idol-nifi/resources/nifi-import-flow.sh [hunk 2]
```diff
@@ -172,7 +172,7 @@ if [ 0 != ${#NEW_PROCESS_GROUP_IDS[@]} ]; then
     SVCSTART_PROCESS_GROUP_IDS=("${NEW_PROCESS_GROUP_IDS[@]}")
     START_PROCESS_GROUP_IDS=("${NEW_PROCESS_GROUP_IDS[@]}")
 
-    for i in {1..12}
+    for ((i=1; i <= ${IDOL_NIFI_SERVICE_START_RETRIES:-12}; i++))
```
Comment (lewis-mackay-mf) [inline helm/idol-nifi/resources/nifi-import-flow.sh:175]: Not sure why it was 12 before, but it does seem like a high value. Does it need to be this high by default?

File: helm/idol-nifi/resources/flow-basic-idol.json (hunk 1 above)
Comment (daniel-lau-mf) [reply helm/idol-nifi/resources/flow-basic-idol.json:79]: I don't really want to stick in values that only adjust the default flow configuration if I can help it.

File: helm/idol-nifi/resources/nifi-import-flow.sh (hunk 2 above)
Comment (daniel-lau-mf) [reply helm/idol-nifi/resources/nifi-import-flow.sh]: It's been 12 from the start (something like 5s sleep = 60s of retry) - reduced to 3 in [c13fe54](https://github.com/opentext-idol/idol-containers-toolkit/pull/97/commits/c13fe548aea3f29129ff819730e9f5864cbf8b84)

**Review by lewis-mackay-mf — APPROVED** [review]

**Pull request merged**